.github/govulncheck/govulncheck-runner.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Run govulncheck security scanning on Kata runtime binaries
+
+set -euo pipefail
+
+# Ensure both Go and go-installed tools are in PATH
+export PATH="/usr/local/go/bin:${HOME}/go/bin:${PATH}"
+
+# Change to runtime directory
+cd ../../src/runtime
+
+binaries=("kata-runtime" "containerd-shim-kata-v2" "kata-monitor")
+declare -A false_positives
+
+# Known false positives (sync with sdl.go FalsePositiveCVEsForBinary)
+# - GO-2025-3595: golang.org/x/net/html - verified not compiled into binary
+# - GO-2025-3488: golang.org/x/oauth2/jws - verified not compiled into binary  
+# - GO-2024-3169: github.com/containers/podman vulnerability not in annotations.go (only constants used)
+# - GO-2024-3042: github.com/containers/podman CVE-2024-3056 not in annotations.go (only constants used)
+# - GO-2023-1962: github.com/containers/podman CVE-2018-10856 not in annotations.go (only constants used) 
+# - GO-2023-1942: github.com/containers/podman CVE-2019-18466 not in annotations.go (only constants used)
+# - GO-2022-1159: github.com/containers/podman CVE-2022-4123 not in annotations.go (only constants used)
+false_positives["containerd-shim-kata-v2"]="GO-2025-3595 GO-2025-3488 GO-2024-3169 GO-2024-3042 GO-2023-1962 GO-2023-1942 GO-2022-1159"
+
+overall_status=0
+
+# Function to filter false positives. This is required as at the moment 
+# there is no native support for silencing vulnerability findings. 
+# See https://go.dev/issue/61211 for updates.
+filter_and_check() {
+  local binary_name="$1"
+  local output="$2"
+  
+  local fp_list="${false_positives[$binary_name]:-}"
+  if [[ -z "$fp_list" ]]; then
+    # No filtering - just check for vulnerabilities
+    if echo "$output" | grep -q "No vulnerabilities found"; then
+      return 0
+    elif echo "$output" | grep -q "GO-\|vulnerability"; then
+      return 1
+    else
+      return 0
+    fi
```

Comment:
```suggestion
    grep -q "GO-\|vulnerability" <<< "$output"
    return $?
```

Reply:
Updated. (the suggested logic was inverted, added with negated grep).